code/game/turfs/turf.dm

```diff
@@ -117,9 +117,11 @@
 
 	var/area/A = loc
 
+	if(A.base_turf)
+		baseturf = A.base_turf
 	if(!baseturf)
```

Comment:
These two conditions seems mutually exclusive? If the area has a baseturf and it is set above, it would follow that there's no need to check the second time if the baseturf is set; the second should probably be an `else if`

Reply:
changed to else if